portality/bll/services/application.py

```diff
@@ -251,6 +252,17 @@ def accept_application(self, application, account, manual_update=True, provenanc
 
         return j
 
+    def reject_update_request_of_journal(self, journal_id):
+        ur, jlock, alock = self.update_request_for_journal(journal_id)
```

Comment:
THis function creates the update request if it does not exist, so you can't use this method here.  You just need to get the UR directly if it exists